plugins/modules/source_control/gitlab/gitlab_runner.py

```diff
@@ -292,10 +302,10 @@ def find_runner(self, description, owned=False):
             # object, so we need to handle both
             if hasattr(runner, "description"):
                 if (runner.description == description):
-                    return self._runners_endpoint.get(runner.id)
+                    return self._gitlab.runners.get(runner.id)
```

Comment:
Similar question here and for the next one. Why not keep using `_runners_endpoint`?

Reply:
Same answer here :-)

Reply:
Bugfix addressed in #3965